consensus/src/lib.rs

```diff
@@ -76,73 +75,120 @@ where
         + 'static,
     Ctx::Future: Send + 'static,
 {
-    let tx_svc = transactions::TxVerifierService::new(database.clone());
-    let block_svc = block::BlockVerifierService::new(ctx_svc, tx_svc.clone(), database);
+    let tx_svc = TxVerifierService::new(database.clone());
+    let block_svc = BlockVerifierService::new(ctx_svc, tx_svc.clone(), database);
     Ok((block_svc, tx_svc))
 }
 
+/// An internal trait used to represent a database so we don't have to write [`tower::Service`] bounds
+/// everywhere.
 pub trait Database:
```

Comment:
"internal" has the idea of being `pub(crate)` or similar, I think a different word would be better here. "Type alias trait" or something.

Could also explain the trait and what it represents.

Reply:
wIll update in the PR where the request.response enums are unified.